services/compute/virtualmachine/virtualmachine.go

```diff
@@ -401,6 +403,11 @@ func (c *client) getWssdVirtualMachineOSConfiguration(s *compute.OSProfile) (*ws
 	if s.CustomData != nil {
 		osconfig.CustomData = *s.CustomData
 	}
+
+	osconfig.ProxyConfiguration, err = c.getWssdVirtualMachineProxyConfiguration(s.ProxyConfiguration, opType)
+	if err != nil {
```

Comment:
How are we enforcing the update of proxy settings is blocked?

Reply:
Arc VM operators, Arc VM cli, and moccli doesnt support update.

Reply:
How can we enforce the same in the moc-sdk?

Reply:
Because the restriction is from Moc(cloudagent/nodeagent) that we dont support moc, I think we should restrict this is moc-sdk or cloudagent

Reply:
I dont see how we can restrict update in moc sdk